sys-devel/clang/clang-11.0.0.ebuild

```diff
@@ -88,6 +92,11 @@ src_prepare() {
 	llvm.org_src_prepare
 
 	mv ../clang-tools-extra tools/extra || die
+
+	# add Gentoo Portage Prefix for Darwin (see prefix-dirs.patch)
+	sed -i -e "s|@GENTOO_PORTAGE_EPREFIX@|${EPREFIX}/|g" \
```

Comment:
please use eprefixify from prefix.eclass to do this:

eprefixify lib/Frontend/InitHeaderSearch.cpp lib/Driver/ToolChains/Darwin.cpp || die

Reply:
done